netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -137,15 +137,16 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
   @Override
   public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) {
     try {
-      if (msg instanceof CreateStreamCommand) {
-        createStream((CreateStreamCommand) msg, promise);
+      if (msg instanceof CreateStreamNettyCommand) {
```

Comment:
Why did you create new classes for these commands? Couldn't we have just changed the existing classes to have a `flush` boolean?